website/src/theme/DocItem/Footer/GitHubContributors.jsx

```diff
@@ -0,0 +1,93 @@
+// biome-ignore lint/correctness/noUnusedImports: Needs this for React to work
+import React, { useState, useEffect } from 'react';
+import { getContributors } from './contributors';
+import styles from './contributors.module.scss';
+
+const unleashTeam = new Map([
+    ['alvinometric', 'developer advocate, Unleash'],
+    ['andreas-unleash', 'senior software engineer, Unleash'],
+    ['chriswk', 'senior software engineer, Unleash'],
+    ['daveleek', 'senior software engineer, Unleash'],
+    ['FredrikOseberg', 'senior software engineer, Unleash'],
+    ['gardleopard', 'senior software engineer, Unleash'],
+    ['gastonfournier', 'senior software engineer, Unleash'],
+    ['ivarconr', 'co-founder, Unleash'],
+    ['kwasniew', 'senior software engineer, Unleash'],
+    ['nnennandukwe', 'developer advocate, Unleash'],
+    ['nunogois', 'senior software engineer, Unleash'],
+    ['sighphyre', 'senior software engineer, Unleash'],
+    ['sjaanus', 'senior software engineer, Unleash'],
+    ['thomasheartman', 'senior software engineer, Unleash'],
+    ['Tymek', 'senior software engineer, Unleash'],
+]);
+
+const GitHubContributors = ({ owner, repo, filePath }) => {
+    const [contributors, setContributors] = useState([]);
+    const url = `https://api.github.com/repos/${owner}/${repo}/commits?path=${filePath}`;
+
+    useEffect(() => {
+        const fetchFileContributors = () => {
+            fetch(url)
+                .then((response) => response.json())
+                .then((commits) => {
+                    console.log(commits);
+                    const contributors = getContributors(commits);
+                    console.log(contributors);
```

Comment:
Oh, we probably wanna remove these logs, though:
```suggestion
                    const contributors = getContributors(commits);
```